cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/ProgramRecord.java

```diff
@@ -16,17 +16,17 @@
 
 package co.cask.cdap.proto.metadata.lineage;
 
-import co.cask.cdap.proto.Id;
+import co.cask.cdap.proto.id.NamespacedEntityId;
 
 import java.util.Objects;
 
 /**
  * Class to serialize program in {@link LineageRecord}.
  */
 public class ProgramRecord {
-  private final Id.NamespacedId entityId;
+  private final NamespacedEntityId entityId;
```

Comment:
This means a change in the responses of HTTP APIs? (due to how the response is serialized)